www/index.php

```diff
@@ -62,6 +62,10 @@
         $conn['plr'] = 0;
     $connectivity['custom'] = $conn;
 }
+$recaptcha = GetSetting['6LfYXTwUAAAAAPyn1ID4UdsXs-ja3P7qPib37asw'];

```

Comment:
No, you "GetSetting" for the name of the setting (like "recaptcha_api_key").  The value in the ini will be the captcha key and should not be hard-coded.

Reply:
thankyou for suggestion